.travis.yml

```diff
@@ -60,10 +60,6 @@ matrix:
 after_success:
   - bash <(curl -s https://codecov.io/bash)
 
-branches:
```

Comment:
Unrelated change?

Reply:
Not entirely:

> This also removes the CI branch restriction so that I could test this feature branch beforehand, otherwise I would have submitted a broken PR the first time!

I also had to make a few more updates to make the tests pass, before cleaning up the commits, and I don't think anyone needs to see all that going on in here. Better to run tests before proposing.

Reply:
I think we want to leave this as it was, this change will cause the `#pypa-dev` channel to get notified about build status for everyone's forks as well.

Are you not able to run the tests locally?

Reply:
> I think we want to leave this as it was, this change will cause the #pypa-dev channel to get notified about build status for everyone's forks as well.

Oh, sorry for the spam I've already triggered!

How about setting it up so forks don't spam the channel? 

One solution is to put the channel name as a secure variable: https://github.com/travis-ci/travis-ci/issues/5063.

---

> Are you not able to run the tests locally?

No, I've tried running tests locally, but it's erroring. It took a very long time to set up and I also don't think I have the disk space for the whole Docker image and all the downloads.

I only want to change some strings, it's quite heavy to do  https://warehouse.readthedocs.io/development/getting-started/#running-tests-and-linters and it downloaded several GB and taken about half an hour doing `make tests`. And then `make serve` is needed because `make tests` isn't working at the moment, which has downloaded several more GB before erroring:

```
ERROR: for warehouse_web_1  Cannot start service web: driver failed programming external connectivity on endpoint warehouse_web_1 (6704063cda248388329eecac5542769d636f0cb3dcf8bdCreating warehouse_worker_1        ... done

ERROR: for web  Cannot start service web: driver failed programming external connectivity on endpoint warehouse_web_1 (6704063cda248388329eecac5542769d636f0cb3dcf8bd3f5a5b3e3e21bda6bc): Error starting userland proxy: Bind for 0.0.0.0:80: unexpected error (Failure EADDRINUSE)
ERROR: Encountered errors while bringing up the project.
make: *** [serve] Error 1
```

An important point of a CI (for me) is to run all the tests in a known clean environment. Much effort has been put into setting up the CI to do all the tests, I'd much rather leave all that work to the CI server. It's also quicker.

It would be great if we could configure the CI so it can run for feature branches in forks without spamming IRC.

Thanks!

Reply:
> One solution is to put the channel name as a secure variable: travis-ci/travis-ci#5063.

I was going to suggest the same -- pip does this.

Reply:
Tests succeed on my local copy, FWIW.

Reply:
@hugovk Can we remove this for now and address it in a separate PR?

Reply:
Sure, will do when I'm next at a keyboard.

Reply:
Done, please see PR https://github.com/pypa/warehouse/pull/3983.